src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -573,46 +592,244 @@ namespace HVACVariableRefrigerantFlow {
 			EMSOverrideHPOperatingMode( false ),
 			EMSValueForHPOperatingMode( 0.0 ),
 			HPOperatingModeErrorIndex( 0 ),
-			RatedEvapCapacity( 40000.0 ),
-			RatedCompPower( 14000.0 ),
+			AlgorithmIUCtrl( 1 ),
 			CondensingTemp( 44.0 ),
+			CondTempFixed( 0.0 ),
+			CoffEvapCap( 1.0 ),
+			CompActSpeed( 0.0 ),
+			CompMaxDeltaP( 0.0 ),
+			C1Te( 0.0 ),
+			C2Te( 0.0 ),
+			C3Te( 0.0 ),
+			C1Tc( 0.0 ),
+			C2Tc( 0.0 ),
+			C3Tc( 0.0 ),
+			DiffOUTeTo( 5 ),
+			EffCompInverter( 0.95 ),
 			EvaporatingTemp( 6.0 ),
+			EvapTempFixed( 0.0 ),
+			HROUHexRatio( 0.0 ),
 			IUEvaporatingTemp( 6.0 ), 
 			IUCondensingTemp( 44.0 ),
 			IUEvapTempLow( 4.0 ),
-			IUEvapTempHigh( 13.0 ),
+			IUEvapTempHigh( 15.0 ),
```

Comment:
Thanks @rraustad; I figured by the name it was unrelated to your model, but good to verify it.